docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -900,6 +901,51 @@ quarkus.oidc.b.tenant-paths=/*/hello <3>
 
 TIP: Path-matching mechanism works exactly same as in the xref:security-authorize-web-endpoints-reference.adoc#authorization-using-configuration[Authorization using configuration].
 
+[[header-based-tenant-resolver]]
+==== Resolve tenants with a token header name
+
+OIDC tenants that support Bearer token authentication can be resolved using the custom HTTP header that contains the token.
+Header-based resolution is enabled when the `quarkus.oidc.token.header` property is set to a value other than `Authorization`. For example:
+
+[source,properties]
+----
+quarkus.oidc.tenant-a.token.header=Custom-Authorization-1   <1>
+quarkus.oidc.tenant-b.token.header=Custom-Authorization-2   <2>
+----
+<1> Tenant `tenant-a` is resolved when the incoming HTTP request contains the `Custom-Authorization-1` header (provided no higher-priority resolver has already matched a tenant).
```

Comment:
```suggestion
<1> Tenant `tenant-a` is resolved when the incoming HTTP request contains the `Custom-Authorization-1` header.
```

We have this message in the docs, so I suggest to drop it as we don't clarify it in other examples either